File: 1-INICIANTE/1151/1151.py
n = int(input())

numeros = [0, 1]
while len(numeros)<n:
  next_value = numeros[-1] + numeros[-2]
  numeros.append(next_value)

for i in numeros:
  if i == numeros[-1]:
    print(f"{i}")
  else: 
    print(f"{i}", end=" ")

File: 1-INICIANTE/1151/1151.rs
use std::io;

fn main(){
  let mut input = String::new();
  io::stdin().read_line(&mut input).expect("Impossível ler entrada.");
  let n: usize = input.trim().parse().expect("Digite um número válido.");

  let mut numeros = vec![0, 1];

  while numeros.len() < n{
    let next_numero = numeros[numeros.len() -1] + numeros[numeros.len() - 2];
    numeros.push(next_numero);
  }

  for (i, numero) in numeros.iter().enumerate(){
    if i == numeros.len() - 1{
      println!("{}", numero);
    }else{
      print!("{} ", numero);
    }
  }
  
}